src/datepicker/datepicker.component.ts

```diff
@@ -141,6 +164,9 @@ export class DatePicker implements OnDestroy {
 		// add day classes and special case the "today" element based on `this.value`
 		Array.from(dayContainer).forEach(element => {
 			element.classList.add("bx--date-picker__day");
+			if (!this.value) {
```

Comment:
Just curious what case this is handling?

Reply:
When there is no assigned value I get a `TypeError: Cannot read property 'length' of undefined`

Reply:
Ah, okay ... maybe a short comment noting that?

Reply:
Well it's a simple function guard so not really sure what to add as a comment